src/map/hash_trie_map/mod.rs

```diff
@@ -458,18 +482,18 @@ mod bucket_utils {
     pub fn list_remove_first<T: Clone, F: Fn(&T) -> bool>(
```

Comment:
Can you remove the rustdoc.  It's outdated but since it doesn't adds any non-obvious information we can just remove it.